Cassiopee/Generator/test/checkCartesian_m1.py

```diff
@@ -77,3 +79,7 @@
     test.testT(t,4)
     print('Is Cartesian::',cartesian,isCartesian)
 #C.convertPyTree2File(t,'t_test4.cgns')
+
+del t
```

Comment:
What do you have in mind? Adding more tests past this line in the future?

Reply:
No.